test/protocol/OfferHandlerTest.js

```diff
@@ -756,7 +755,7 @@ describe("IBosonOfferHandler", function () {
           ).to.revertedWith(RevertReasons.INVALID_DISPUTE_RESOLVER);
         });
 
-        it("Dispute resolver is not active", async function () {
+        it.skip("Dispute resolver is not active", async function () {
```

Comment:
maybe add a `// TODO - revisit when account deactivations are supported` above the test.

Reply:
done